app/views/layouts/_main_nav.html.erb

```diff
@@ -1,6 +1,10 @@
-<% nav_links = NavigationLink.all.ordered.to_a %>
+<% default_nav_links = NavigationLink.default_section.ordered.to_a %>
+<% other_nav_links = NavigationLink.other_section.ordered.to_a %>
 
 <nav class="mb-6 <% unless user_signed_in? %>mt-4<% end %>" aria-label="Primary sidebar nav">
+  <div class="mb-1 fw-bold">
+    <%= community_name %>
+  </div>
```

Comment:
small accessibility issue (blocking) - I think this and the "Other" should be level 2 headings (which might involve tweaking the styles to keep the same visual appearance). This will help build the "document outline" of the page, and let assistive tech users jump straight to these sections if they're looking for them 🙂